x/payments/spec/03_messages.md

```diff
@@ -0,0 +1,145 @@
+
+## MsgCreatePaymentContract 
+
+The owner of a bond can edit some of the payment's parameters using `MsgCreatePaymentContract`.
+
+| **Field**              | **Type**           | **Description**                                                                                               |
+|:-----------------------|:-------------------|:--------------------------------------------------------------------------------------------------------------|
+| CreatorDid             | `did.Did`          | Did of the user
+| PaymentTemplateId      | `string`           | ID of the paymentTemplate
+| PaymentContractId      | `string`           | ID of the PaymentContract
+| Payer                  | `sdk.AccAddress`   | Address of the payer
+| CanDeauthorise         | `bool`             | Bool of de_authorise
+| DiscountId             | ` sdk.Uint`        | Any discount given
+
+```go
+type MsgCreatePaymentContract struct {
+	CreatorDid        did.Did        `json:"creator_did" yaml:"creator_did"`
+	PaymentTemplateId string         `json:"payment_template_id" yaml:"payment_template_id"`
+	PaymentContractId string         `json:"payment_contract_id" yaml:"payment_contract_id"`
+	Payer             sdk.AccAddress `json:"payer" yaml:"payer"`
+	CanDeauthorise    bool           `json:"can_deauthorise" yaml:"can_deauthorise"`
+	DiscountId        sdk.Uint       `json:"discount_id" yaml:"discount_id"`
+}
+```
+This message is expected to fail if:
+- any editable field violates the restrictions set for the same field in `MsgCreatePaymentContract`
+- all editable fields are `"[do-not-modify]"`
+
+## MsgCreateSubscription 
+
+The owner of a bond can edit some of the bond's parameters using `MsgCreateSubscription`.
+
+| **Field**              | **Type**           | **Description**                                                                                               |
+|:-----------------------|:-------------------|:--------------------------------------------------------------------------------------------------------------|
+| CreatorDid             | `did.Did`          | Did of the user 
+| SubscriptionId         | `string`           | ID for the subscription
+| PaymentContractId      | `string`           | ID for the paymentContract
+| MaxPeriods             | `sdk.AccAddress`   | 
+| Period                 | `bool`             | IF the periods are allowed or not
+
+
+```go
+type MsgCreateSubscription struct {
+	CreatorDid        did.Did  `json:"creator_did" yaml:"creator_did"`
+	SubscriptionId    string   `json:"subscription_id" yaml:"subscription_id"`
+	PaymentContractId string   `json:"payment_contract_id" yaml:"payment_contract_id"`
+	MaxPeriods        sdk.Uint `json:"max_periods" yaml:"max_periods"`
+	Period            Period   `json:"period" yaml:"period"`
+}
+```
+This message is expected to fail if:
+- any editable field violates the restrictions set for the same field in `MsgCreateSubscription`
+- all editable fields are `"[do-not-modify]"`
```

Comment:
This line is from bonds module and does not apply here

Reply:
removed the part